modules/compactor/config.go

```diff
@@ -31,6 +31,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 
 	f.DurationVar(&cfg.Compactor.BlockRetention, util.PrefixConfig(prefix, "compaction.block-retention"), 14*24*time.Hour, "Duration to keep blocks/traces.")
 	f.IntVar(&cfg.Compactor.MaxCompactionObjects, util.PrefixConfig(prefix, "compaction.max-objects-per-block"), 6000000, "Maximum number of traces in a compacted block.")
+	f.Uint64Var(&cfg.Compactor.MaxBlockSize, util.PrefixConfig(prefix, "compaction.max-block-size"), 100*1024*1024*1024 /* 100GB */, "Maximum size of a compacted block.")
```

Comment:
`max-block-bytes` ?  I think we've been ending all the new "bytes" config options with the unit.

Reply:
Sounds good, this also matches the setting added in #474